src/hound/config/config_test.go

```diff
@@ -0,0 +1,36 @@
+package config
```

Comment:
Heads up @jklein, if we commit an invalid config-example.json the tests will break.